src/Request/RequestHandler.php

```diff
@@ -428,6 +441,9 @@ protected function sortParameters(array $parameters)
 
     /**
      * Escapes all special symbols for query
+     * All parameter names and values are escaped using the percent-encoding (%xx) mechanism.
+     * Characters not in the unreserved character set ([RFC3986] section 2.3) MUST be encoded.
+     * Characters in the unreserved character set MUST NOT be encoded.
```

Comment:
@darrynten Actually all special characters are properly encoded, thanks to rawurlencode so there is absolutely no security problem.